src/challenge/views.py

```diff
@@ -289,6 +291,17 @@ def post(self, request):
             if challenge.first_blood is None:
                 challenge.first_blood = user
                 challenge.save()
+                hook = os.getenv("BLOOD_HOOK")
+                if hook:
+                    chall_clean = challenge.name.replace('`', '')
```

Comment:
Couple of formatting things, imo `challenge_clean` over `chall_clean` but thats not a big deal, also the project style is to only use `"`, this can be resolved automatically by running `poetry run make format`.